src/pages/people/tabs/Wanted.spec.tsx

```diff
@@ -520,4 +526,113 @@ describe('Wanted Component', () => {
       }
     });
   });
+
+  test('Test that org bounties are scrollable on org home page', async () => {
+    const createdMockBounties = Array.from({ length: 30 }, (_: any, index: number) => ({
+      ...(mockBounties[0] || {}),
+      bounty: {
+        ...(mockBounties[0]?.bounty || {}),
+        id: mockBounties[0]?.bounty?.id + index + 1
+      }
+    }));
+
+    const userBounties = createdMockBounties.map((bounty: any, index: number) => ({
+      ...bounty,
+      body: {
+        ...bounty.bounty,
+        owner_id: person.owner_pubkey,
+        title: `This is a test bounty ${index}`
+      }
+    })) as any;
+
+    (usePerson as jest.Mock).mockImplementation(() => ({
+      person: {},
+      canEdit: true
+    }));
+
+    (useStores as jest.Mock).mockReturnValue({
+      main: {
+        getPersonCreatedBounties: jest.fn(() => [userBounties])
+      },
+      ui: {
+        selectedPerson: '123',
+        meInfo: {
+          owner_alias: 'test'
+        }
+      }
+    });
+
+    jest
+      .spyOn(mainStore, 'getPersonCreatedBounties')
+      .mockReturnValue(Promise.resolve(userBounties));
+    act(async () => {
+      const { getByText, getAllByTestId } = render(
+        <MemoryRouter initialEntries={['/p/1234/bounties']}>
+          <Route path="/p/:uuid/bounties" component={Wanted} />
+        </MemoryRouter>
+      );
+      await waitFor(() => getAllByTestId('user-created-bounty'));
+      fireEvent.scroll(window, { target: { scrollY: 1000 } });
+
+      expect(getByText('Load More')).toBeInTheDocument();
+    });
+  });
+
+  test('Test that post a bounty button, auto populates newer bounties on org bounty page and bounties section on profile', async () => {
+    const userBounty = { ...createdBounty, body: {} } as any;
+    userBounty.body = {
+      ...userBounty.bounty,
+      owner_id: person.owner_pubkey,
+      title: 'new text',
+      description: 'new text'
+    };
+
+    (usePerson as jest.Mock).mockImplementation(() => ({
+      person: {},
+      canEdit: true
+    }));
+
+    jest.spyOn(mainStore, 'getOrganizationBounties');
+    act(async () => {
+      const { getByText } = render(
+        <MemoryRouter initialEntries={['/p/1234/bounties']}>
```

Comment:
@okhot rendering the Wanted Component here means you are testing the user-created bounties page here, and not The Org Page.